src/components/atoms/resize-handle/ResizeHandle.tsx

```diff
@@ -0,0 +1,4 @@
+import React from "react";
+import X from Y;
+
+export const Handle = () => {};
```

Comment:
Implement a resize handle, consumable by a https://github.com/react-grid-layout/react-grid-layout#custom-child-components-and-draggable-handles